db/seeds.rb

```diff
@@ -41,14 +43,31 @@
   'Townhouse Apts'       => 30
 }
 
+# if this array changes then the logic below has to also change
+directions = ['North', 'South']
+
 stops.each do |route_number, stop_names|
-  stop_names.each do |stop_name|
-    # Anytime in the last two months
-    Timecop.freeze rand(86_400).minutes.ago do
-      stop = BusStop.find_or_initialize_by name: stop_name
-      stop.hastus_id = hastus_ids.fetch(stop_name)
-      stop.routes << routes.fetch(route_number)
-      stop.save!
+  directions.each do |direction|
+    if direction == 'North'
+      sequence = 0
+    else
+      sequence = stop_names.length
```

Comment:
Can someone please explain why `stop_name.length`? is this arbitrary?

Reply:
The algorithm counts down from the length of the array for southbound stops.

Reply:
Edit: not a bus driver. Don't come at me for having stops that aren't in the same route, guys :)

Reply:
Sorry dave didn't see your comment

Reply:
I see, so `stop_name` is an array containing the stops? Not the actual name of the stop?

Reply:
'stop_names' is an array containing the stop names. If you look at the `each` loop, it's initially using `stops` (defined [here](https://github.com/umts/stop-project/pull/80/files/e4d0ddb5c023e8c21c1361a6dece852737165ce9#diff-f7a25bc30a409fc77050703fc048ca8cL21)).